src/mono/mono/metadata/sysmath.c

```diff
@@ -206,12 +206,6 @@ ves_icall_System_Math_Log2 (gdouble x)
 	return log2 (x);
 }
 
-gdouble
-ves_icall_System_Math_ScaleB (gdouble x, gint32 n)
```

Comment:
@vargaz, I have noticed that in some PRs, instead of deleting the now-unused icall, the function is wrapped in `#ifndef ENABLE_NETCORE`. Should we do the same here for https://github.com/mono/mono/pull/20424?

Reply:
If the icall is used for framework Mono we want to ifdef it out, as the code is mirrored to `mono/mono` and still required there. In this case the icall was netcore-specific, so getting rid of it entirely is preferable.

On a related note, this was a nice reminder to merge the corresponding mono/mono PR :)